src/components/containers/FoldEmpty.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+
+export default class FoldEmpty extends Component {
+  render() {
+    const {children, icon: Icon, messagePrimary, messageSecondary} = this.props;
+
+    return (
+      <div className="fold__content__empty">
+        <div className="fold__content__empty__icon">
+          {Icon ? <Icon /> : null}
+        </div>
+        <div className="fold__content__empty__message__primary">
+          {messagePrimary ? messagePrimary : null}
+        </div>
+        <div className="fold__content__empty__message__secondary">
+          {messageSecondary ? messageSecondary : null}
+        </div>
```

Comment:
I tend to prefer to have the conditional on the `<div...` itself, because often the div's will have padding on them or some other layout modifier, and we'd want to remove that if the content is gone, too